### ISSUE TITLE
incrRegEx is undefined in v1.0.4

### ISSUE THREAD

**Issue by aashutoshrathi** [nurulc/incr-regex-package#9]
In the package version 1.0.4, as listed as the latest one here: https://www.npmjs.com/package/incr-regex-package
[PS: The commits on master are only till 1.0.3]

I tried this code, which throws `incrRegEx is not a function.`
```js
const ir = require("incr-regex-package");
const { incrRegEx } = ir;
```

Downgrading to v1.0.3 works BTW.

So I dug deeper later and saw in node_modules that in v1.0.4, there is no export for `incrRegEx`, which is literally the heart of this package.

Just wanted to know if this package is being actively maintained (and is it safe to use), and also did someone accidentally pushed /published their test code as v1.0.4 to npm, without ever committing here?

Thanks!

**Comment by Fasteroid** [comment]
I think it's dead bro

**Comment by aashutoshrathi** [comment]
yeah seems dead, I don't even remember anymore what I was working on when I stumbled upon this

**Comment by Fasteroid** [comment]
> yeah seems dead, I don't even remember anymore what I was working on when I stumbled upon this

~~I have plans to rewrite this package if you're still in need!~~

~~I'm currently designing a feature like vscode's command pallete, but smarter, so you can pass arguments to the commands via the search bar.~~

~~The only way I can think of to do this efficiently is via describing the shape of each option with a regex... but then how do you do partial matches?~~

EDIT: [peggy](https://peggyjs.org/index.html)

**Comment by TomStrepsil** [comment]
> yeah seems dead, I don't even remember anymore what I was working on when I stumbled upon this

You might want to look at [`regex-partial-match`](https://github.com/TomStrepsil/regex-partial-match), which translates a regular expression into a partially-matching one (with some caveats).

**Comment by Fasteroid** [comment]
https://github.com/Fasteroid/partial-pattern-tree
